pkg/features/operator_features.go

```diff
@@ -51,4 +55,5 @@ func init() {
 var defaultOperatorFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	PartitionedUpdate: {Default: false, PreRelease: featuregate.Alpha},
 	Decommission:      {Default: false, PreRelease: featuregate.Alpha},
+	Upgrade:           {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
@alinadonisa can we deco Upgrade and Enable the others?  I do not want to push a PR that will conflict with yours